services/ui-src/src/components/export/ExportedEntityDetailsOverlaySection.tsx

```diff
@@ -196,9 +169,14 @@ const sx = {
     textAlign: "center",
     paddingBottom: "5rem",
   },
-  entityInformation: {
-    padding: "1rem 0 1rem 0",
-    fontWeight: "bold",
+  stepName: {
+    fontSize: "18px",
+    paddingBottom: "0.75rem",
+  },
+  stepHint: {
+    fontSize: "16px",
+    fontWeight: "normal",
+    color: "#5B616B",
```

Comment:
```suggestion
  stepName: {
    fontSize: "lg",
    paddingBottom: "0.75rem",
  },
  stepHint: {
    fontSize: "md",
    fontWeight: "normal",
    color: "#5B616B",
```

Reply:
I'm starting to see the #5B616B crop up in a couple places. We should define it in the colors -> palette custom section in the theme.ts file